### DISCUSSION TITLE
Postgressuite and ItemsRoute intermittent failures

### DISCUSSION THREAD

**Discussion by gitplaneta** [gvolpe/pfps-shopping-cart#642] [General]
If you change `shop.generators.nonEmptyStringGen` to generate shorter strings, ex. to:
```
  val nonEmptyStringGen: Gen[String] =
    Gen
      .chooseNum(1, 40)
      .flatMap { n =>
        Gen.buildableOfN[String, Char](n, Gen.alphaChar)
      }
```
`PostgresSuite` will more likely fail with unique constraint violations and `ItemsRuiteSuite:items by brand succeeds` will fail because of `toLowerCase.capitalize` in `BrandParam.toDomain` string manipulation.

**Comment by gvolpe** [comment]
That's not an issue. If you change the generators, you need to adapt the test to work with such constraints. The more general approach would be to have a generator that guarantees *unique* brand names, if you insist on changing the current basic one.

**Reply by gitplaneta** [reply]
Yep, that's a minor thing but the above tests will fail if ran enough times even without changing the generators.
